### ISSUE TITLE
Improve accessibility of Warp AI view

### ISSUE THREAD

**Issue by dteare** [warpdotdev/Warp#3648]
### Describe the solution you'd like?

```Text
I have poor vision and am unable to read the text shown in the Warp UI view without using the macOS zoom feature.
```

### Is your feature request related to a problem? Please describe.

```Text
I require larger fonts than anyone would prefer. I'd love to be able to enjoy higher information density, but it's simply not possible for me.

I really like the ⌘+ option you provide for making the terminal commands and output larger. It's perfect. The challenge I have is the Warp UI view doesn't follow suit. 

In the attached screenshot you can see the terminal window font is larger than the ai one. It might seem like the Warp UI font is not _that_ much smaller, but it's very difficult for me to read. In fact, as the day goes on and my eyes get tired, I will need to bump the console font up and the ai window will be completely unreadable without relying on the macOS zoom feature. Zooming in, however, has it's own challenges. 

Being able to move Warp UI into its own window and allow it to have it's own customizable font size would be lovely.
```

### Additional context

<img width="1022" alt="CleanShot 2023-09-09 at 11 53 27@2x" src="https://github.com/warpdotdev/Warp/assets/110676/0aea8014-1963-4fea-8ee7-4f25e0fcf90e">

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by elviskahoro** [comment]
@dteare This is a great call out. Thanks for bringing this to our attention.

**Comment by cameronbarker** [comment]
I just brought this up in this [discord thread](https://discord.com/channels/851854972600451112/1155922282817003541). If there is any way as a community member I could support, I'd be happy to work on a PR.

**Comment by ryota-murakami** [comment]
What's matter about this?
I'm web dev and change font size acomplished just edit 1 line css on web development.

Change font size is big technical challenge in the Warp that Rust based application?

**Comment by dannyneira** [comment]
Hi folks, we'll be tracking this issue over as a duplicate of #1443 

ALso regarding the WARP AI text, we're working on improvements which would allow the text to scale correctly with the terminal text so stay tuned.

**Comment by dannyneira** [comment]
Duplicate of https://github.com/warpdotdev/Warp/issues/1443
